public/src/components/epicTests/newNameCreator.tsx

```diff
@@ -50,7 +50,9 @@ class NewNameCreator extends React.Component<NewNameCreatorProps, NewNameCreator
   }
 
   isDuplicateName = (newName: string) => {
-    const isDuplicate = this.props.existingNames.includes(newName);
+    const newLowerCase: string = newName.toLowerCase();
+    const existingLowerCase: string[] = this.props.existingNames.map(name => name.toLowerCase());
+    const isDuplicate = existingLowerCase.includes(newLowerCase);
```

Comment:
This is fine, but it can be slightly more efficient (only requires up to 1 pass through the array):
```
const newLowerCase: string = newName.toLowerCase();
const isDuplicate = this.props.existingNames.some(name => name.toLowerCase() === newLowerCase);
```

Reply:
👍